src/apps/patron-page/sections/ReservationDetailsSection.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, FC, useEffect } from "react";
+import { PatronV5 } from "../../../core/fbs/model";
+import CheckBox from "../../../components/checkbox/Checkbox";
+import BranchesDropdown from "../util/BranchesDropdown";
+import { useText } from "../../../core/utils/text";
+import DateInputs from "../../../components/date-inputs/date-inputs";
+
+interface ReservationDetailsSectionProps {
+  patron: PatronV5;
+  changePatron: (newValue: string | boolean, key: string) => void;
```

Comment:
The changePatron type is defined in multiple places. I think it should only be defined once for better maintenance and clarity.

Reply:
fixed